managed/ui/src/components/config/Storage/StorageConfiguration.js

```diff
@@ -149,22 +164,44 @@ class StorageConfiguration extends Component {
         break;
     }
 
-    return this.props
-      .addCustomerConfig({
-        type: 'STORAGE',
-        name: type,
-        data: dataPayload
-      })
-      .then((resp) => {
-        if (getPromiseState(this.props.addConfig).isSuccess()) {
-          // reset form after successful submission due to BACKUP_LOCATION value is shared across all tabs
-          this.props.reset();
-          this.props.fetchCustomerConfigs();
-        } else if (getPromiseState(this.props.addConfig).isError()) {
-          // show server-side validation errors under form inputs
-          throw new SubmissionError(this.props.addConfig.error);
-        }
-      });
+    if (values.type === "update") {
+      this.setState({ enableEdit: false });
+      return this.props
+        .updateCustomerConfig({
+          type: 'STORAGE',
+          configUUID: values.configUUID,
+          name: type,
+          data: dataPayload
+        })
+        .then((resp) => {
+          if (getPromiseState(this.props.addConfig).isSuccess()) {
+            // reset form after successful submission due to BACKUP_LOCATION value is shared across all tabs
+            this.props.reset();
+            this.props.fetchCustomerConfigs();
+          } else if (getPromiseState(this.props.addConfig).isError()) {
+            // show server-side validation errors under form inputs
+            throw new SubmissionError(this.props.addConfig.error);
```

Comment:
must be `props.updateConfig` everywhere above

Reply:
Done.